### PR TITLE
[BS-7] add assets from UIKit

### PR DESCRIPTION
**Pull request by antonhoang**
export all asset into pdf

### CODE REVIEW

File: bonsai/Appearance/Appearance.swift [hunk 1]
```diff
@@ -8,6 +8,19 @@
 import Foundation
 import SwiftUI
 
+enum BonsaiFont {
+    static let bonsai_34 = UIFont.systemFont(ofSize: 34, weight: .init(rawValue: 700))
+    static let bonsai_28 = UIFont.systemFont(ofSize: 28, weight: .init(rawValue: 700))
+    static let bonsai_22 = UIFont.systemFont(ofSize: 22, weight: .init(rawValue: 700))
```
Comment (VladimirBrejcha) [inline bonsai/Appearance/Appearance.swift]: naming is not very clear, could we name these better?
I like how `body_17` and others sound

File: bonsai/Assets.xcassets/Category/cross.vial.fill.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "images" : [
+    {
+      "filename" : "cross.vial.fill.pdf",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "cross.vial.fill-1.pdf",
+      "idiom" : "universal",
+      "scale" : "2x"
```
Comment (VladimirBrejcha) [inline bonsai/Assets.xcassets/Category/cross.vial.fill.imageset/Contents.json]: for pdf we can simply specify universal or single scale

**Comment by VladimirBrejcha** [comment]
Just a quick reminder, that for sf symbols icons we don't need to import any assets, instead we can use UIImage(systemName: <sf symbol name>)

**Comment by antonhoang** [comment]
> Just a quick reminder, that for sf symbols icons we don't need to import any assets, instead we can use UIImage(systemName: )

should we use https://github.com/SFSafeSymbols/SFSafeSymbols for swiftgen?

File: bonsai/Appearance/Appearance.swift (hunk 1 above)
Comment (antonhoang) [reply bonsai/Appearance/Appearance.swift]: maybe main_34, main_28 ?

File: bonsai/Assets.xcassets/Category/cross.vial.fill.imageset/Contents.json (hunk 2 above)
Comment (antonhoang) [reply bonsai/Assets.xcassets/Category/cross.vial.fill.imageset/Contents.json]: ok

**Comment by VladimirBrejcha** [comment]
Just checked their sources, seems they still force unwrap all the symbols, so it is not really 'safe '

https://github.com/SFSafeSymbols/SFSafeSymbols/blob/7a2310d55391f063e8a31b9d587d593d1dd3af9c/Sources/SFSafeSymbols/Initializers/UIKit/UIImageExtension.swift#L13

I think we can come up with simple symbol namespace (as we do with fonts) with force unwraps for system images

File: bonsai/Appearance/Appearance.swift (hunk 1 above)
Comment (VladimirBrejcha) [reply bonsai/Appearance/Appearance.swift]: What I meant is that `body`, `title`, `caption` etc, they are expressive, while `main` or `bonsai` doesn't give any understanding where to use the font

**Review by VladimirBrejcha — APPROVED** [review]

**Review by Valter4578 — APPROVED** [review]

**Pull request merged**